.env.sample

```diff
@@ -1,9 +1,12 @@
 MONGO_URI=mongodb://localhost:27017/db
 REDIS_URI=redis://localhost:6379
 
+TEST_MONGO_URI=mongodb://localhost:27017/testdb
```

Comment:
Can we setup a ci workflow here?

Reply:
Yes, I'll send you a test redis and mongo uri